lbz/lambdas/broker.py

```diff
@@ -0,0 +1,40 @@
+from typing import Callable, Dict
+
+from lbz.exceptions import LambdaFWException
+from lbz.handlers import BaseHandler
+from lbz.lambdas.enums import LambdaResult
+from lbz.lambdas.response import LambdaResponse, lambda_error_response
+from lbz.misc import get_logger
+
+logger = get_logger(__name__)
+
+
+class LambdaBroker(BaseHandler):
+    def __init__(
+        self,
+        mapper: Dict[str, Callable[..., LambdaResponse]],
+        event: dict,
+        context: object = None,
+    ) -> None:
+        self.event = event
+        self.context = context
+        self.mapper = mapper
+
+    def handle(self) -> LambdaResponse:
+        if not (op := self.event.get("op")):
+            logger.error('Missing "op" field in the processed event: %r', self.event)
+            return lambda_error_response(LambdaResult.BAD_REQUEST, 'Missing "op" field in event.')
```

Comment:
As the final word, I think this should be `CONTRACT_ERROR` as this will be used for internal communication. Here, we are both client and server, so, when something like this failed, it means we did not meet the contract between services (lambdas):
```suggestion
            return lambda_error_response(LambdaResult.CONTRACT_ERROR, 'Missing "op" field.')
```